### ISSUE TITLE
Club Faculty Advisor

### ISSUE THREAD

**Issue by medensqui** [PlanHubMe/PlanHub#22]
This could be Mr. Neiers but if he doesn't/can't do it, who should we ask? @thatoddmailbox

**Comment by thatoddmailbox** [comment]
@medensqui IDKLOL!

Maybe a compsci person? I'm not really sure...

**Comment by medensqui** [comment]
@thatoddmailbox True we could ask comp sci people
Or just someone who is willing to be the advisor but it would be helpful to have someone from NLTL/CompSci

**Comment by thatoddmailbox** [comment]
@medensqui IDK if/when the compsci people are available.

**Comment by willbarkoff** [comment]
@medensqui @thatoddmailbox Am I allowed to be in the club even though I am in middle school.

**Comment by medensqui** [comment]
@nywillb Well we're meeting Thursday 12:50 - 1:35 so if you have class then no :cry: 
However, we can send out weekly newsletter to everyone because not everyone will be able to come to every meeting @thatoddmailbox

**Comment by thatoddmailbox** [comment]
@nywillb I don't think you have a matching free time, but you can still contribute outside of the club.
